src/main/java/com/github/invader/agent/interceptors/PeakProfile.java

```diff
@@ -0,0 +1,29 @@
+package com.github.invader.agent.interceptors;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.time.LocalTime;
+import java.util.List;
+
+/**
+ * Definition of a peak profile.
+ * @see PeakInterceptor
+ *
+ * Created by Jacek on 2017-07-01.
+ */
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class PeakProfile {
+
+    private LocalTime startTime;
```

Comment:
Consider changing this to OffsetTime, or whatever seems appropriate, so that time zones are also taken into account. This will be useful when trying to coordinate multiple services spread across different time zones.